collections/triple.go

```diff
@@ -273,26 +273,19 @@ func (t tripleKeyCodec[K1, K2, K3]) SizeNonTerminal(key Triple[K1, K2, K3]) int
 	return size
 }
 
-func PrefixTripleRange[K1, K2, K3 any](k1 K1) Ranger[Triple[K1, K2, K3]] {
+// NewPrefixUntilTripleRange defines a collection query which ranges until the provided Pair prefix.
+// Unstable: this API might change in the future.
+func NewPrefixUntilTripleRange[K1, K2, K3 any](k1 K1) Ranger[Triple[K1, K2, K3]] {
```

Comment:
This commit https://github.com/cosmos/cosmos-sdk/pull/17144/commits/5973ad40ed3e2240dfbf653d039dd3e41df9d0f3, made this one inconsistent (cc @atheeshp). Could you align their naming?

Reply:
I actually followed the pair key.
https://github.com/cosmos/cosmos-sdk/blob/main/collections/pair.go#L219-L232